Dockerfile.builder

```diff
@@ -40,11 +40,25 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
         ccache \
         libelf-dev \
         python3-docutils \
-        pahole \
         libcap-dev \
         ${CLANG} \
         llvm-${CLANG_VERSION} \
         lld \
         kmod \
         rsync \
-        libc6-dev-i386
+        libc6-dev-i386 \
+        cmake \
+        libdw-dev \
+        git
+
+RUN cd /tmp && \
+    git clone https://git.kernel.org/pub/scm/devel/pahole/pahole.git && \
+    cd pahole && \
+    git checkout v1.27 && \
```

Comment:
To ease maintenance, it would be nice to have an `ARG PAHOLE_VERSION=1.27` somewhere higher up in the file, close to the clang version. This way, it's less buried and will stand out more to whomever bumps clang next time.

Reply:
Yea, sounds like a good idea